data/gallery/balustrades.ts

```diff
@@ -1,22 +1,107 @@
+import image1 from "public/gallery/balustrades/1.jpg";
```

Comment:
I changed it to staticly loaded images. After build nextjs will already know image sizes etc. so it will handle them better. I forgot previously that it's a server side rendering xD